packages/taiko-client/internal/testutils/helper.go

```diff
@@ -282,6 +282,50 @@ func SignatureFromRSV(r, s string, v byte) []byte {
 	return append(append(hexutil.MustDecode(r), hexutil.MustDecode(s)...), v)
 }
 
+func SendDynamicFeeTxWithNonce(
```

Comment:
Seems this function name doesn't match what it will do? maybe `assembleTestTx()`?